src/flags.go

```diff
@@ -87,7 +88,7 @@ func intialiseFlags() {
 
 	flags.StringVarP(&CurrentConfig.DataText, "data-text", "d", "",
```

Comment:
Let's rename `data-text` to simply `data` or `data-text-or-file` to clarify the new meaning of this argument.

Reply:
Yep, done.